spatialpy/Result.py

```diff
@@ -224,6 +378,55 @@ def get_property(self, property_name, timepoints=None):
             ret = ret.flatten()
         return ret
 
+    def plot_property(self, property_name, t_ndx, title=None, return_plotly_figure=False):
+        """ Plots the Results using plotly. Can only be viewed in a Jupyter Notebook.
+
+            If concentration is False (default), the integer, raw, trajectory data is returned,
+            if set to True, the concentration (=copy_number/volume) is returned.
+
+            If deterministic is True, show results for determinstic (instead of stochastic) values
+
+        Attributes
+        ----------
+        property_name : str
+            A string describing the property to be plotted.
+        t_ndx : int
+            The time index of the results to be plotted
+        title : str
+            The title of the graph
+        return_plotly_figure : bool
+            whether or not to return a figure dictionary of data(graph object traces) and layout options
+            which may be edited by the user.
+        """
+        from plotly.offline import init_notebook_mode, iplot
```

Comment:
Need to add plotly to the requiements.txt file

Reply:
Just added